packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```
@@ -173,9 +173,659 @@
+              },
+              {
+                "$ref": "#/components/schemas/GrantPermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RemovePeerRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RemoveSignatoryRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/RevokePermissionRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SetAccountQuorumRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/SubtractAssetQuantityRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/TransferAssetRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/CompareAndSetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetBlockRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetSignatoriesRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetPendingTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetTransactionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountAssetsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAccountDetailRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetAssetInfoRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetRolePermissionsRequestParameters"
+              },
+              {
+                "$ref": "#/components/schemas/GetEngineReceiptsRequestParameters"
+              }
+            ]
+          }
+        }
+      },
+      "AddAssetQuantityRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Add Asset Quantity.",
+        "type": "object",
+        "required": [
+          "assetId",
+          "amount"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          },
+          "amount": {
+            "type": "integer"
```

amount can be double: 200.02, so I suggest to restore:
```suggestion
            "type": "number"
```